.github/workflows/test-hexpm-mirrors.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: test-hexpm-mirror
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test_hexpm_mirror_option:
+    name: Usage of hexpm-mirror option
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'windows-latest']
```

Comment:
I'd use something "not latest" here. Otherwise, when the target finally moves, in a few months/years, it is possible the tests start failing, as explained in [this comment](https://github.com/erlef/setup-beam/issues/155#issuecomment-1314502264).